app/helpers/sessions_helper.rb

```diff
@@ -0,0 +1,19 @@
+module SessionsHelper
+  def current_user
+    if (user_id = session[:user_id])
```

Comment:
Bỏ ngoặc, `user_id == session[:user_id]`

Reply:
chỗ này e gán biến ko phải so sánh ạ

Reply:
https://stackoverflow.com/questions/1966585/is-assignment-in-a-conditional-clause-good-ruby-style